### PR TITLE
Fix biopet 607

### PR DESCRIPTION
**Pull request by rhpvorderman**
Autogenerated some missing documentation.

### CODE REVIEW

File: biopet-autodoc/synchronize_docs.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
```
Comment (sndrtj) [inline biopet-autodoc/synchronize_docs.sh:2]: For some extra checking, you may want to set `set -eu` and `set -o pipefail` in the beginning of the script. This makes sure bash ends when it encounters an error or an unset variable.

File: biopet-autodoc/templateparser.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+from jinja2 import Environment, FileSystemLoader
+import os
+import argparse
+import ast
+import yaml
+
+if __name__ == "__main__":
+    
+    #Block with command line options
+    parser=argparse.ArgumentParser(description="")
```
Comment (sndrtj) [inline biopet-autodoc/templateparser.py:13]: The `description` is an optional variable. If you set it to an empty string, you may as well leave it out altogether ;-).

File: biopet-autodoc/templateparser.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    parser.add_argument("-t", type=str, nargs=1, help="Templatefile to be used")
```
Comment (sndrtj) [inline biopet-autodoc/templateparser.py:14]: If you don't set `nargs`, accessing this value becomes easier, as you don't have to access a list.

File: biopet-autodoc/templateparser.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    tool['run']=arguments.tool_run
+    help_file=open(arguments.tool_option_list, 'r')
+    tool['option_list']=help_file.read()   #This reads the option list file to a raw string
+
+    # Open jinja2 template and render it with variables
+    j2_env = Environment(loader=FileSystemLoader("/"))
+    rendered = j2_env.get_template(template).render(tool=tool)
+
+    # Write the template results to the output file
+    file=open(output_file, "w")
```
Comment (sndrtj) [inline biopet-autodoc/templateparser.py:37]: The `seek` call here is unnecessary. Also, if you put this under a context handler, you do not have to manually close the handle.
E.g.:

```
with open(output_file, "w") as output_handle:
    output_handle.write(rendered)
```

**Review by sndrtj — CHANGES_REQUESTED** [review]
Absolutely great idea! :+1: 

I do have some small comments on the scripts. Please check them out :-).

**Comment by rhpvorderman** [comment]
The autodocumentation will be done in another way.

**Pull request closed without merging**